typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
 
 
 # get the directory path to this file; 
-LIBS_DIR = os.path.dirname(os.path.abspath(__file__))
-PLUGIN_DIR = os.path.dirname(LIBS_DIR)
-PACKAGES_DIR = os.path.dirname(PLUGIN_DIR)
+LIBS_DIR = dirname(dirname(os.path.abspath(__file__)))
+PLUGIN_DIR = dirname(LIBS_DIR)
+PACKAGES_DIR = dirname(PLUGIN_DIR)
```

Comment:
This doesn't handle if my plugin is installed via a symlink. For example, on my machine this points to '/Users/bill/src' - which is clearly not what is intended.  Why not just use `sublime.packages_path()`, which on my machine is defined as `/Users/bill/Library/Application Support/Sublime Text 2/Packages`